src/main/resources/templates/features/update.html

```diff
@@ -7,6 +7,10 @@
 
 <div th:replace="fragments :: page-header"></div>
 
+<h3 th:text="${'Update Feature ' + {feature.id}}"></h3>
+<h4 th:text="${errorMsg}" th:if="${errorStatus}"></h4>
```

Comment:
This is the other half of the validation for the update pages. When errorStatus is true (when an error is found in the controller), the error message is shown.